helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -27,19 +27,18 @@ data:
         loadbalance {{ .Values.loadbalancePolicy }}
         prometheus :{{ .Values.ports.prometheus }}
 
-        {{- if .Values.configmap.proxy }}
-        {{- range (.Values.configmap.proxy | trimAll "\n " |  split "\n") }}
-        proxy {{ . }}
-        {{- end }}
+        {{- if .Values.configmap.forward }}
+        forward . {{ range (.Values.configmap.forward | trimAll "\n " |  split "\n") }}{{ . }}{{ end }} /etc/resolv.conf
+        {{- else }}
+        forward . /etc/resolv.conf
         {{- end }}
 
-        {{- if .Values.configmap.custom }}
-        {{- range (.Values.configmap.custom | trimAll "\n " |  split "\n") }}
-        {{ . }}
-        {{- end }}
-        {{- end }}
-
-        proxy . /etc/resolv.conf
         reload
         autopath @kubernetes
     }
+
+    {{- if .Values.configmap.custom }}
+    {{- range (.Values.configmap.custom | trimAll "\n " |  split "\n") }}
+    {{ . }}
+    {{- end }}
+    {{- end }}
```

Comment:
Now there is no possibility to use more than one forward instance per server block. So I am moving the custom block out of the catch-all main block so customers have more flexibility. I will add it to the docs